vnext/src-win/Libraries/Modal/Modal.d.ts

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
If this is an override of a file from core, it should be added to the overrides.json file